NGitLab.Tests/PackageTests.cs

```diff
@@ -0,0 +1,33 @@
+using System.Threading.Tasks;
+using NGitLab.Models;
+using NGitLab.Tests.Docker;
+using NUnit.Framework;
+
+namespace NGitLab.Tests
+{
+    public class PackageTests
+    {
+        [Test]
+        [NGitLabRetry]
+        public async Task Test_publish_package()
+        {
+            using var context = await GitLabTestContext.CreateAsync();
+            var project = context.CreateProject();
+            var packagesClient = context.Client.Packages;
+
+            var packagePublish = new PackagePublish
+            {
+                ProjectId = project.Id,
+                FileName = "../../../../README.md",
+                PackageName = "Packages",
+                PackageVersion = "1.0.0",
+                Status = "default",
+            };
+
+            var package1 = packagesClient.Publish(packagePublish);
+
+            // TODO: What assertions should be put here?
+            Assert.True(true);
```

Comment:
You should implement https://docs.gitlab.com/ee/api/packages.html#get-a-project-package, so you can fetch the package after publishing it and assert the package is available